src/pages/pricing/pricing.constants.ts

```diff
@@ -0,0 +1,41 @@
+export const pricingData = [
+  {
+    type: 'Free',
+    pricePerMonth: '0',
+    buttonText: 'Get started',
+    btnSubtitle: 'For developers building apps, with no creator restrictions',
+    listTitle: 'Features',
```

Comment:
if this is constant, then why put it in json? You can directly put it in tsx right?